### DISCUSSION TITLE
I want to use ESPAsyncWebServer library in Arduino as a component.

### DISCUSSION THREAD

**Discussion by M-Rizwan-Anwar** [espressif/arduino-esp32#9291] [Q&A]
I am trying to add library "ESPAsyncWebServer" and "AsyncTCP" in Arduino component, I have added them in Arduino All libraries like this:

set(ARDUINO_ALL_LIBRARIES
  ArduinoOTA
  AsyncUDP
  AsyncTCP               // This One I added
  BLE
  BluetoothSerial
  DNSServer
  ESPAsyncWebServer // This One I added
  EEPROM
  ESP_I2S
  ESP_SR
.
.
.
and also I've added sources like this:
set(ARDUINO_LIBRARY_ESPAsyncWebServer_SRCS
  libraries/ESPAsyncWebServer-mastersrc/AsyncEventSource.cpp
  libraries/ESPAsyncWebServer-mastersrc/AsyncWebSocket.cpp
  libraries/ESPAsyncWebServer-mastersrc/SPIFFSEditor.cpp
  libraries/ESPAsyncWebServer-mastersrc/WebAuthentication.cpp
  libraries/ESPAsyncWebServer-mastersrc/WebHandlers.cpp
  libraries/ESPAsyncWebServer-mastersrc/WebRequest.cpp
  libraries/ESPAsyncWebServer-mastersrc/WebResponses.cpp
  libraries/ESPAsyncWebServer-mastersrc/WebServer.cpp
  libraries/ESPAsyncWebServer-mastersrc/AsyncTCP.cpp)

  set(ARDUINO_LIBRARY_AsyncTCP_SRCS
  libraries/ESPAsyncWebServer-mastersrc/AsyncTCP.cpp)

but I am getting the error " Cannot Find ESPAsyncWebServer,h" and same for "AsyncTCP.h"

**Comment by SuGlider** [comment]
This seems duplicated with https://github.com/espressif/arduino-esp32/discussions/9296.
